submissions/crealime/dom-api/js/main.js

```diff
@@ -0,0 +1,149 @@
+document.addEventListener("DOMContentLoaded", function() {
+
+  const url = 'json/movies.json' // Json file with movies data
+  let moviesData = [] // Storage for movies data
+  const menuUl = document.querySelector('.menu__ul')
+  const menu = document.querySelector('.menu')
+  const movieContent = document.querySelector('.main')
+  const movieRandomize = document.querySelector('.menu__random')
+  const menuShowButton = document.querySelector('.hamburger')
+
+  // Fill menu template with data
+  function menuTemplate({id, title}) {
```

Comment:
Name of function must be a verb. In this case better will be `getMenuTemplate` (example)

Reply:
Done.